doc/source/mast_data_products/hap-parameters.rst

```diff
@@ -3,10 +3,46 @@ HAP Parameters
 
 .. _hap-parameters:
 
+The HAP parameter files are housed in the drizzlepac/pars/hap_pars directory. There are different json parameter files for the SVM and MVM products in separate directories. The svm_parameter json files are also used for pipeline products.
+
+
+In either directory, there is a separate detector json file for each instrument/detector. Each file specifies additional json files used for different tasks and scenarios. The primary differences are in the case of different number of exposures:
```

Comment:
Please reword some of this text to be more general.  What I write here is only an EXAMPLE to give you a sense of what I am trying to describe.  You can use your own words.

In the high-level svm_parameters or mvm_parameters directories, the instrument/detector-dependent configuration index files  (e.g., acs_sbc_index.json) provide a map to the configuration files used for the major HAP processing steps.

For the astrodrizzle step, there are different files which should be employed depending upon the number of exposures int the visit.